src/frontend/src/service/api-service.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+type MethodOptions =
+  | 'POST'
+  | 'post'
+  | 'GET'
+  | 'get'
+  | 'DELETE'
+  | 'delete'
+  | 'head'
+  | 'HEAD'
+  | 'put'
+  | 'PUT'
+  | 'patch'
+  | 'PATCH';
+
+type Req = {
```

Comment:
I think we should rename `Req` to a full word, so it's clear whether this is "requirements" or "request". https://github.com/airbnb/javascript#naming--descriptive

Reply:
I have updated req to request.  Thank you for the article, I will keep this in mind.